src/components/donation/Steps/DonateMethods/ChariotConnect/Form.tsx

```diff
@@ -0,0 +1,144 @@
+  const methods = useForm<FormValues>({
+    defaultValues: details || initial,
+  });
+
+  const currency = methods.watch("currency");
+
+  return (
+    <FormProvider {...methods}>
+      <form
+        onSubmit={methods.handleSubmit((fv) =>
+          dispatch(
+            setDetails({
+              ...fv,
+              method: "chariot",
+            })
+          )
+        )}
+        className="grid gap-4"
+      >
+        <CurrencySelector
+          currencies={[USD_CURRENCY]}
+          disabled={methods.formState.isSubmitting}
+          label="Currency"
+          // only one currency available, so can't change it
+          onChange={() => {}}
+          value={currency}
+          classes={{ label: "font-semibold" }}
+          required
+        />
+        <Field<FormValues>
+          name="amount"
+          label="Donation amount"
+          classes={{ label: "font-semibold" }}
+          required
+          // validation must be dynamicly set depending on which exact currency is selected
+          registerOptions={{
+            required: "required",
+            min: currency.min
+              ? {
+                  value: currency.min,
+                  message: `must be greater than ${currency.min}`,
+                }
+              : undefined,
+            pattern: {
+              value: /[0-9]+/,
+              message: "must be a number",
+            },
+            shouldUnregister: true,
+          }}
+          tooltip="The minimum donation amount will depend on which DAF provider you select in the next step."
+        />
+        {!authUserEmail && (
+          <Field<FormValues>
+            name="email"
+            label="Email"
+            required
+            classes={{ label: "font-semibold" }}
+            registerOptions={{
+              required: "required",
+              validate: (value) =>
+                requiredString.email().isValidSync(value) || "invalid email",
+            }}
+          />
+        )}
+        {!recipient.isKYCRequired && (
+          // if KYC is required, the checkbox is redundant
+          <CheckField<FormValues>
+            name="userOptForKYC"
+            classes={{
+              container: "text-sm",
+              error: "mt-2",
+            }}
+          >
+            Please send me a tax receipt
+          </CheckField>
+        )}
+        <AdvancedOptions
+          display={advanceOptDisplay}
+          splitComponent={
+            <Split<FormValues, "pctLiquidSplit">
+              className="mb-6"
+              liqPctField="pctLiquidSplit"
+            />
+          }
+        />
+        <p className="text-sm text-gray-d2 dark:text-gray mt-4">
+          Please click the button below and follow the instructions provided to
+          complete your donation
+        </p>
+
+        <button
+          disabled={methods.formState.isSubmitting}
+          className="btn-orange btn-donate mt-2"
+          type="submit"
+        >
+          <LoadText
```

Comment:
might not need load text, as submission is just synchronous

Reply:
Removed `disabled` prop assignments from all */DonateMethods/\*/Form* files as they have basically the same logic.